services/coincap/coincap-changepercent24hr.service.js

```diff
@@ -0,0 +1,42 @@
+import Joi from 'joi'
+import { floorCount } from '../color-formatters.js'
+import BaseCoincapService from './coincap-base.js'
+
+const schema = Joi.object({
+  data: Joi.object({
+    changePercent24Hr: Joi.string().required(),
```

Comment:
If we're going to treat this as a number in `percentFormat()`, we should validate that it is a number in the schema.